data_override/README.MD

```diff
@@ -10,26 +10,25 @@
 
 #### 1. YAML Data Table format:
 Each entry in the data_table has the following key values:
-- **gridname:** Name of the grid to interpolate the data to. The acceptable values are "ICE", "OCN", "ATM", and "LND"
-- **fieldname_code:** Name of the field as it is in the code to interpolate.
-- **fieldname_file:** Name of the field as it is writen in the file. **Required** only if overriding from a file
-- **file_name:** Name of the file where the variable is located, including the directory. **Required** only if overriding from a file
-- **interpol_method:** Method used to interpolate the field. The acceptable values are "bilinear", "bicubic", and "none". "none" implies that the field in the file is already in the model grid. The LIMA format is no longer supported. **Required** only if overriding from a file
+- **grid_name:** Name of the grid to interpolate the data to. The acceptable values are "ICE", "OCN", "ATM", and "LND"
+- **fieldname_in_model:** Name of the field as it is in the code to interpolate.
+- **override_file:** The parent key containing the nested keys related to overriding data from file. **Required** only if overriding from a file
+  - **file_name:** Name of the file where the variable is located, including the directory
+  - **fieldname_in_file:** Name of the field as it is writen in the file
+  - **interp_method:** Method used to interpolate the field. The acceptable values are "bilinear", "bicubic", and "none". "none" implies that the field in the file is already in the model grid. The LIMA format is no longer supported
+  - **multi_file:** The multifile parent key. **Required** only if it is desired to use multiple(3) input netcdf files instead of 1. Note that **file_name** must be the second file in the set when using multiple input netcdf files
+    - **prev_file_name:** The name of the first file in the set
+    - **next_file_name:** The name of the third file in the set
+  - **external_weights:** The external weights parent key. **Required** only if it is desired to use external weights from file
```

Comment:
I noticed the only supported method for external weights is bilinear.  Is the `interp_method` used in conjunction with the `external_weights` to determine the method?

It seems we may be opening things up to undetermined behavior using the `interp_method` in conjunction with the `external_weights`.  Is there a method or some other global attribute in the file we could use to compare against the chosen `interp_method`?

Reply:
> Is there a method or some other global attribute in the file we could use to compare against the chosen interp_method

The only thing that we currently do is check that the weight file has the expected dimensions for the interp method. It can be done more rigorous though.